evm-contracts/src/v0.6/examples/VRFD20.sol

```diff
@@ -0,0 +1,112 @@
+pragma solidity 0.6.6;
+
+import "../VRFConsumerBase.sol";
+
+/**
+ * @notice A Chainlink VRF consumer which uses randomness to mimic the rolling
+ * of a 20 sided die
+ */
+contract VRFD20 is VRFConsumerBase {
+    using SafeMathChainlink for uint;
+
+    struct Roll {
```

Comment:
There is no reason to keep the request ID around, that is really just internal plumbing details. I think we should not expose it in the example, as it may give the impression to other people that they need to expose it with the randomness as well.